atom-publisher-lib/src/main/scala/com/gu/atom/data/DataStore.scala

```diff
@@ -24,7 +24,6 @@ trait DataStore extends DataStoreResult {
   /* this will only allow the update if the version in atom is later
    * than the version stored in the database, otherwise it will report
    * it as a version conflict error */
-  def updateAtom(newAtom: Atom): DataStoreResult[Unit]
 
```

Comment:
why was this moved into `(Preview|Published)DataStore` traits? Is it to cover the fact that if you just have a data store you can't update an atom, you need to know whether you have a preview or a published data store?